### ISSUE TITLE
unknown support for ios chrome

### ISSUE THREAD

**Issue by cdaringe** [browserslist/browserslist#643]
# Problem

By studying https://github.com/browserslist/browserslist#browsers, it is unknown if browserslist supports ios chrome.

With proper coaching, I can send a patch to the README to help clarify.

[chrome ios mobile installation URL](https://apps.apple.com/us/app/google-chrome/id535886823).

<details>
<summary>Example user agent + parse output</summary>

<pre>
uap('Mozilla/5.0 (iPhone; CPU iPhone OS 10_3 like Mac OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) CriOS/56.0.2924.75 Mobile/14E5239e Safari/602.1')
{
  ua: 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_3 like Mac OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) CriOS/56.0.2924.75 Mobile/14E5239e Safari/602.1',
  browser: { name: 'Chrome', version: '56.0.2924.75', major: '56' },
  engine: { name: 'WebKit', version: '602.1.50' },
  os: { name: 'iOS', version: '10.3' },
  device: { vendor: 'Apple', model: 'iPhone', type: 'mobile' },
  cpu: { architecture: undefined }
}
</pre>
</details>

**Comment by ai** [comment]
We do not support iOS Chrome.

To add browser support we need to add it to Can I Use, we just use their DB.

Why it is unknown if there is no iOS Chrome in the list?

**Comment by cdaringe** [comment]
Thanks :)

> Why it is unknown if there is no iOS Chrome in the list?

Perhaps it's feasible that iOS Chrome is categorized _within_  the Chrome or ios_safari browser labels. A peer of mine suggested that at some point, ios forced other browser vendors to use essentially use safari webviews at one point in time. So perhaps ios chrome _reported as safari_ could have been legitimate? I'm not stating this with confidence :).  I just want to learn if ios chrome _is_ actually hidden/merged the current data, or knowingly, fully unknown or omitted from caniuse.  I tried going to https://gs.statcounter.com/ which caniuse says is the backing datasource, but I was able to discern the answer to this question. However, it
